src/Nest/Indices/IndexManagement/CloneIndex/CloneIndexRequest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Nest
+{
+	[MapsApi("indices.clone")]
+	[ReadAs(typeof(CloneIndexRequest))]
+	public partial interface ICloneIndexRequest
+	{
+		[DataMember(Name ="aliases")]
+		IAliases Aliases { get; set; }
+
+		[DataMember(Name ="settings")]
+		IIndexSettings Settings { get; set; }
+	}
+
+	public partial class CloneIndexRequest
+	{
+		public CloneIndexRequest(IndexName index, IndexName target, IIndexState state) : this(index, target)
```

Comment:
Should we support this overload, @Mpdreamz, @codebrain ?

Reply:
That's a though one, I rather not since constructors are for path parameters only.

An alternative solution would be to expose `RemoveReadOnlySettings()` as an extension method on `IIndexSettings` and `IndexState`.

Reply:
OK, I'll remove this overload for now